tests/publiccloud/ipa.pm

```diff
@@ -186,6 +186,21 @@ EOT
     );
 }
 
+sub gcp_run_ipa {
```

Comment:
Hi,
when we now touch this test again. My idea was to change it using the provider classes.

```
my $provider = $self->{provider} = $self->provider_factory();
$provider->init();
my $instance = $provider->ipa(
        instance_type => get_var('PUBLIC_CLOUD_INSTANCE_TYPE'),
        cleanup       => 1,
        image_id   => get_var('PUBLIC_CLOUD_IMAGE_ID'),
        tests => get_required_var('PUBLIC_CLOUD_IPA_TESTS')
);
$provider->cleanup();
```
The implementation of launching ipa tool, needs to be done in lib/publiccloud/google.pm then. 

Don't know if it is still the focus of this PR, as the name only mention fixing the upload.

Reply:
not for this PR but I will take that into consideration for the next one when I implement ipa in google. Thanks!